src/main/java/team9/ddang/chat/service/ChatRoomServiceImpl.java

```diff
@@ -41,9 +41,11 @@ public ChatRoomResponse createChatRoom(ChatRoomCreateServiceRequest request, Mem
         Member opponentMember = findMemberByIdOrThrowException(request.opponentMemberId());
 
         List<Member> members = new ArrayList<>();
-        members.add(currentMember);
         members.add(opponentMember);
 
+        List<Member> curmembers = new ArrayList<>();
+        members.add(currentMember);
```

Comment:
```suggestion
        curmebers.add(currentMember);
```

여기 이거 하시려고 하신 거 아닌가요??

Reply:
찾아주셔서 감사합니다... 삽질 재대로 했네요...😅